lib/ansible/plugins/lookup/cyberarkpassword.py

```diff
@@ -144,9 +144,9 @@ def get(self):
                     if "passprops" not in result_dict:
                         result_dict["passprops"] = {}
                     output_prop_name = output_names[i][10:]
-                    result_dict["passprops"][output_prop_name] = output_values[i]
+                    result_dict["passprops"][output_prop_name] = output_values[i].decode("utf-8")
```

Comment:
Please use `to_native` from `from ansible.module_utils._text import to_native`

Reply:
Thank you, fixed.